test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
Damn, we're sending mixed signals now :-/

@viktorklang I suggested to create a unit test, because this is pure standard library testing, so it doesn't need partest. See https://github.com/scala/scala-dev/issues/295 and https://github.com/scala/scala-partest/issues/75.